tests/test_debug/test_accumulation_table.py

```diff
@@ -478,7 +480,41 @@ def test_output_to_existing_file(tmp_path) -> None:
     shutil.rmtree(tmp_path)
 
 
-def test_output_to_new_file(tmp_path) -> None:
+def test_csv_output_to_existing_file(tmp_path) -> None:
```

Comment:
You can improve the quality of these tests by using [pytest fixtures](https://docs.pytest.org/en/stable/how-to/fixtures.html) to define the common elements. Note that you can parametrize fixtures (described on that webpage) to pass either `'table'` or `'csv'`.